bin/build-phar.sh

```diff
@@ -1,37 +1,6 @@
 #!/usr/bin/env bash
-if ! php -r 'extension_loaded("bz2") or exit(1);' ; then
-	echo "You need to install (or enable) bz2 php extension"
-	exit 1
-fi
+composer bin box install
 
-composer global require 'humbug/php-scoper:^1.0@dev'
```

Comment:
this has been replaced by `bamarni/composer-bin-plugin`, up to you if you're cool with that or you prefer the `composer global` or maybe switch to a `wget` the Box PHAR